docs/API-functions.md

```diff
@@ -557,6 +558,14 @@ where:
 * `result`: result of the operation (if this operation recently updated this structure)
 * `srcaddr`: address to which `id` should be bound
 * `peeraddr`: address to which `id` should be connected
+* `priority`: priority for backup group
+
+The priority is set to 0 by default by `srt_prepare_endpoint()` - you can set
+it to a different value afterwards. The default 0 value is the highest priority
+and greater values declare lower priorities. The priority for the backup
+groups decide which link to activate first when the currently active link is
+unstable and which should keep transmitting when multiple active links are
```

Comment:
groups determines which link is activated first when the currently active link is
unstable, and which should keep transmitting when multiple active links are